tools/tpm2_createpolicy.c

```diff
@@ -180,6 +176,12 @@ static bool on_option(char key, char *value) {
     case 'e':
         pctx.common_policy_options.extend_policy_session = true;
         break;
+    case 'C':
+        pctx.common_policy_options.context_file = value;
+        if (value == NULL || value[0] == '\0') {
+            return false;
+        }
```

Comment:
value == NULL impossible and an empty string, who cares, no need for this check.